src-java/reroute-topology/reroute-storm-topology/src/main/java/org/openkilda/wfm/topology/reroute/service/RerouteService.java

```diff
@@ -98,21 +99,24 @@ public void rerouteAffectedFlows(MessageSender sender, String correlationId, Rer
 
         for (FlowWithAffectedPaths entry : groupPathsForRerouting(affectedFlowPaths)) {
             Flow flow = entry.getFlow();
-            transactionManager.doInTransaction(() -> {
-                updateFlowPathsStateForFlow(switchId, port, entry.getAffectedPaths());
+            Boolean sendRerouteRequest = transactionManager.doInTransaction(() -> {
+                boolean flowPathFound = updateFlowPathsStateForFlow(switchId, port, entry.getAffectedPaths());
```

Comment:
swapping this and following lines you can get rid of `flowPathFound` variable.

Reply:
I can't swap lines because flow status computation is dependent on flow paths statuses.